eZ/Publish/API/Repository/Tests/SearchServiceLocationTest.php

```diff
@@ -209,6 +210,35 @@ public function testNonPrintableUtf8Characters(): void
         );
     }
 
+    /**
+     * @covers \eZ\Publish\API\Repository\SearchService::findLocations
+     */
+    public function testEscapedNonPrintableUtf8Characters(): void
+    {
+        $setupFactory = $this->getSetupFactory();
+        if ($setupFactory instanceof Legacy) {
```

Comment:
so does it work for Elasticsearch 1.x then?

Reply:
It should, as from what I see Elasticsearch Search Engine uses Field value mappers - if not, I think that's another issue (as it should).
Also, I changed the condition in `if` as every `SetupFactory` is instance of `Legacy` - this should work better now.